support-frontend/assets/components/ticker/contributionTicker.jsx

```diff
@@ -36,7 +38,7 @@ const percentageTotalAsNegative = (total: number, goal: number) => {
 
 
 // ----- Component ----- //
-export class ContributionTicker extends Component<PropTypes, StateTypes> {
+export default class ContributionTicker extends Component<PropTypes, StateTypes> {
```

Comment:
Maybe we should document work TBD in the class somewhere. Like: (1) we'd like to edit the component to get its values either from JSON or with props and (2) add some documentation about the structure of the JSON the component expects.

Reply:
I've actually documented those in this Trello card instead: https://trello.com/c/qXxukzxY/395-ticker-in-storybook-improvements-for-future-campaigns